charts/spire/charts/spire-server/values.yaml

```diff
@@ -155,19 +155,19 @@ persistence:
 
 dataStore:
   sql:
-    ## @param dataStore.sql.databaseType Other supported databases are "postgres" and "mysql"
+    ## @param dataStore.sql.databaseType Other supported databases are ["postgres", "mysql", "aws_postgresql", "aws_mysql"]
```

Comment:
Can we add a note that these are experimental as they havent been tested yet.

Reply:
Done